code/hud/hud.cpp

```diff
@@ -1178,7 +1178,9 @@ bool HudGauge::canRender()
 	}
 	
 	if (render_for_cockpit_toggle > 0) {
-		if (cockpitActive && (render_for_cockpit_toggle == 2)) {
+		if (((Viewer_mode & (VM_CHASE)) != 0) && (render_for_cockpit_toggle == 2)) {
```

Comment:
You can simplify by simply putting `(Viewer_mode & VM_Chase)` instead of comparing to zero and putting VM_Chase in parentheses.

Reply:
The earlier part of the code a few lines up uses a `== 0` check for a bit, and in theory that could be simplified too, but is possibly beyond the scope of this PR.

Reply:
Done